onnxruntime/python/tools/transformers/run_benchmark.sh

```diff
@@ -20,6 +20,7 @@ run_install=true
 run_ort=true
 run_torch=false
 run_torchscript=true
+run_tensorflow=false
```

Comment:
Not integrate ORT benchmark using onnx exporting from tf in this script because those onnx(from tf) only works with 3 inputs